api/python/cellxgene_census/src/cellxgene_census/experimental/pp/_online.py

```diff
@@ -56,6 +56,33 @@ def finalize(
         return batches_u, batches_var, all_u, all_var
 
 
+class MeanAccumulator:
```

Comment:
I agree. I implemented batches (although only partially) mostly because the code was already there, and this could be useful for future usages. But I agree with your statement. If Pablo doesn't think that batches are useful, it's probably worth simplifying this to use numpy as you suggest.

Reply:
Simplified as requested.